src/main/java/com/techsophy/tsf/account/service/impl/ACLServiceImpl.java

```diff
@@ -99,18 +101,32 @@ public ACLSchema getACLById(String id)
     public ACLValidate checkACLAccess(String id, CheckACLSchema checkACLSchema) throws JsonProcessingException, AccessDeniedException
     {
         Map<String,Object> userDetailsFromKeycloak= tokenUtils.getUserInformationMap(tokenUtils.getTokenFromContext());
+        AuditableData auditableData = userFormDataService.getUserFormDataByUserId(userDetailsFromKeycloak.get("userId")+"",false);
+        UserFormDataSchema userFormDataSchema = objectMapper.convertValue(auditableData,UserFormDataSchema.class);
+        RequestProperties requestProperties = new RequestProperties();
+        requestProperties.setUserInfo(userDetailsFromKeycloak);
+        requestProperties.setContext(checkACLSchema.getContext()+"");
```

Comment:
please test before committing and pushing the code, this is a string here and a map there